pkg/flow/flowtest/flowtest.go

```diff
@@ -0,0 +1,195 @@
+// Package flowtest provides a script-based testing framework for Flow.
+//
+// [TestScript] accepts a path to a txtar file, where the comment of the txtar
+// file denotes the script to run. Files in the txtar archive are unpacked to a
+// temporary directory as part of the test.
+//
+// See [rsc.io/script] for more information about the scripting language and
+// available default commands.
+//
+// # Flow commands
+//
+// In addition to the default commands provided by [rsc.io/script], the
+// following Flow-specific commands are available:
+//
+//   - `controller.load [file]`: Load a file into the Flow controller.
+//   - `controller.start`: Start the Flow controller.
+//   - `assert.health [component] [health]`: Assert the health of a specific component.
```

Comment:
I think we should just point to the file that registers all these commands. This is likely to get outdated fast.